src/lib/tabs/tab-body.ts

```diff
@@ -88,13 +139,16 @@ export type MdTabBodyOriginState = 'left' | 'right';
     ])
   ]
 })
-export class MdTabBody implements OnInit, AfterViewChecked {
+export class MdTabBody implements OnInit {
   /** The portal host inside of this container into which the tab body content will be loaded. */
-  @ViewChild(PortalHostDirective) _portalHost: PortalHostDirective;
+  @ViewChild(MdTabBodyPortal) _portalHost: MdTabBodyPortal;
 
   /** Event emitted when the tab begins to animate towards the center as the active tab. */
   @Output() onCentering: EventEmitter<number> = new EventEmitter<number>();
 
+  /** Event emitted when the tab completes its animation away from the center. */
+  @Output() afterLeavingCenter: EventEmitter<number> = new EventEmitter<number>();
```

Comment:
Can we prefix this w/ a underscore since its private ... so `_afterLeavingCenter`?

Whats the `number` type used for?

Reply:
Done, and the number was the result of a copy paste from the emitter above.  Since I wasn't using the result at all I didn't notice.